instrumentation/src/javatests/org/oppia/android/instrumentation/topic/TopicActivityTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.android.instrumentation.topic
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.Before
+import org.junit.Test
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.findObjectByRes
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.findObjectByText
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.scrollRecyclerViewTextIntoView
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.startOppiaFromScratch
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.waitForRes
+import org.oppia.android.instrumentation.testing.EndToEndTestHelper.waitForText
+
+/** Tests for TopicActivity. */
+class TopicActivityTest {
```

Comment:
I think this should be "TopicTest" instead.

Lets wait for ben's suggestion

Reply:
Renamed to TopicTest for now. Will revert this if Ben suggests otherwise.